### ISSUE TITLE
Add SLAS Admin API

### ISSUE THREAD

**Issue by hnestmann** [SalesforceCommerceCloud/sfcc-ci#231]
We now have the shopper login and api access API, which can be administered via API. 

https://developer.commercecloud.com/s/api-details/a003k00000VzoEyAAJ/commerce-cloud-developer-centershopperloginandapiaccessadmin

I suggest the following commands
```
    slas:tenant:add [options]                                                                                   
    slas:tenant:get [options]                                                                                   
    slas:tenant:delete [options]

    slas:client:add [options]
    slas:client:get [options]
    slas:client:delete [options]
    slas:client:replace [options]
```

**Comment by johnboxall** [comment]
Here is an example mini-CLI for interacting with the SLAS API: https://gist.github.com/johnboxall/5fc961a6b6a13440a9a6afd37264459c

**Comment by tobiaslohr** [comment]
Released with v2.8.0
